articles/ai-services/translator/document-translation/how-to-guides/create-sas-tokens.md

```diff
@@ -96,6 +96,9 @@ Go to the [Azure portal](https://portal.azure.com/#home) and navigate to your co
 
 1. To [construct a SAS URL](#use-your-sas-url-to-grant-access), append the SAS token (URI) to the URL for a storage service.
 
+++ Note: SAS cannot be generated on the folder-level as Flat-namespace blob containers don't have real folders, and 
+   don't support folder-level security. 
```

Comment:
```suggestion
> [!NOTE]
> SAS can't be generated on the folder level because Flat-namespace blob containers don't have real folders and don't support folder-level security. 
```